libcontainer/dmz/dmz_linux.go

```diff
@@ -1,4 +1,5 @@
-//go:build !runc_nodmz
```

Comment:
I don't think we need to add this here -- if we build runc-dmz then we can use it. I will take this along with @rata's suggestion and make a PR to add MIPS binaries to our release artifacts (presumably Debian has the necessary cross-compilers).

Reply:
> add MIPS binaries to our release artifacts

I don't think we have to do this, MIPS is no longer official on our side.

Reply:
Can we merge this, and discuss your concern in a separate PR?